net/ddns-scripts/files/update_cnkuai_cn.sh

```diff
@@ -0,0 +1,83 @@
+# inside url we need domain, username and password
```

Comment:
Please run through shellcheck.

Reply:
> Please run through shellcheck.

That's awesome tool. You let me know it, thank you.